ui/tempo-plugin/src/plugins/tempo-trace-query/DashboardTempoTraceQueryEditor.tsx

```diff
@@ -45,7 +45,6 @@ export function DashboardTempoTraceQueryEditor(props: DashboardTempoTraceQueryEd
         />
       </FormControl>
       <TraceQLEditor
-        completeConfig={{ remote: { url: datasourceURL } }}
```

Comment:
We are using the eslint linter. If you can add eslint to your IDE on save it would save you some work :)